pkg/network/node/networkpolicy.go

```diff
@@ -71,8 +71,9 @@ type npPolicy struct {
 	watchesAllPods    bool
 	watchesOwnPods    bool
 
-	flows       []string
-	selectedIPs []string
+	flows         []string
+	selectedIPs   []string
+	matchesAllIPs bool
```

Comment:
sorry, didn't see your question yesterday until just now.. so actually, maybe "`selectsAllIPs`" would be better, to go with `selectedIPs`?

Reply:
Sounds good. Done.